.bazelci/examples.yml

```diff
@@ -142,6 +142,8 @@ tasks:
     name: "Scala example"
     platform: ubuntu1804
     working_directory: examples/scala_akka
+    shell_command: 
```

Comment:
Please check out the project locally and re-pin it, and include the updated maven_install.json in this PR.

Reply:
Done.